app/src/main/java/com/kabouzeid/gramophone/provider/MusicPlaybackQueueStore.java

```diff
@@ -67,7 +67,7 @@ private void createTable(@NonNull final SQLiteDatabase db, final String tableNam
         builder.append("(");
 
         builder.append(BaseColumns._ID);
-        builder.append(" INT NOT NULL,");
+        builder.append(" LONG NOT NULL,");
```

Comment:
Not sure if there needs to be more changes around this - from what I understand SQLite doesn't actually really care about typing information but it may be best to update the type on upgrade.

Reply:
Just checked, onUpgrade just rebuilds the table structure - I incremented the version number to put these changes into play.